### PR TITLE
Fix disable_grub_timeout for older version of SLE

### PR DESCRIPTION
**Pull request by mimi1vx**
http://quasar.suse.cz/tests/1101#step/disable_grub_timeout/3

### CODE REVIEW

File: tests/installation/disable_grub_timeout.pm [hunk 1]
```diff
@@ -15,6 +15,7 @@ use strict;
 use warnings;
 use base "y2logsstep";
 use testapi;
+use utils;
 
 sub run {
```
Comment (SergioAtSUSE) [inline tests/installation/disable_grub_timeout.pm]: I would save the hotkey in a var at that point:

```
if (check_var('DISTRI', 'sle') && sle_version_at_least('12-SP2') || check_var('DISTRI', 'opensuse')) {
  var bootloader_hotkey = 'alt-r';
}
else {
  var bootloader_hotkey = 'alt-t';
}
```

Notice: && has precedence over ||, so you don't need extra parentheses.

File: tests/installation/disable_grub_timeout.pm [hunk 2]
```diff
@@ -30,13 +31,26 @@ sub run {
...
+            wait_screen_change { send_key 'alt-r'; };
+        }
+        else {
+            wait_screen_change { send_key 'alt-t'; };
+        }
+    }
+    # openSUSE all supported distributions use 'alt-r'
+    else {
+        wait_screen_change { send_key 'alt-r'; };
+    }
```
Comment (SergioAtSUSE) [inline tests/installation/disable_grub_timeout.pm]: And here simply one:

```
wait_screen_change { send_key $bootloader_hotkey; };
```

File: tests/installation/disable_grub_timeout.pm (hunk 1 above)
Comment (mimi1vx) [reply tests/installation/disable_grub_timeout.pm]: weird , I updated my branch with simuliar bit simpler change .. and here isnt update visible

Comment (SergioAtSUSE) [reply tests/installation/disable_grub_timeout.pm]: Yes, I also have a broken PR for that reason. It seems that github observer for new push'es to a branch is currently broken.

I had closed my old PR and did the PR again. It did work so. Could you try?

**Comment by SergioAtSUSE** [comment]
Now I am seeing the changes

**Pull request merged**